RiotSwiftUI/Modules/Room/PollTimeline/Coordinator/PollTimelineCoordinator.swift

```diff
@@ -0,0 +1,143 @@
+// File created from SimpleUserProfileExample
+// $ createScreen.sh Room/PollTimeline PollTimeline
+/*
+ Copyright 2021 New Vector Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+import UIKit
+import SwiftUI
+import MatrixSDK
+import Combine
+
+struct PollTimelineCoordinatorParameters {
+    let session: MXSession
+    let room: MXRoom
+    let pollStartEvent: MXEvent
+}
+
+@available(iOS 14.0, *)
+final class PollTimelineCoordinator: Coordinator, PollAggregatorDelegate {
+    
+    // MARK: - Properties
+    
+    // MARK: Private
+    
+    private let parameters: PollTimelineCoordinatorParameters
+    private let selectedAnswerIdentifiersSubject = PassthroughSubject<[String], Never>()
+    
+    private var pollAggregator: PollAggregator
+    private var pollTimelineViewModel: PollTimelineViewModel!
```

Comment:
Wondering why one is implicitly unwrapped and the other isn't as they're both set in init?

Reply:
Because it would be using self before all stored properties are initialized.

Reply:
Ah yep with the `buildTimelinePollFrom` method, gotcha.